src/controller/modules_fn.py

```diff
@@ -106,11 +106,19 @@ def addImages(widget):
     """
     compute addition of all input images
     """
-    parent_names = getParentNames(widget)
-    for parent_name in parent_names:
-        if parent_name not in _IMAGES_STACK.keys():
-            return print("'{}' not in image stack".format(parent_name))
-    im = core.addImages([_IMAGES_STACK[parent_name] for parent_name in parent_names])
+    if widget.singleValue.isChecked():
+        try:
+            value = float(widget.value.text())
+        except ValueError:
+            return print("addition value ERROR")
+        ref_parent_name = widget.reference.currentText()
+        im = core.substractImages(_IMAGES_STACK[ref_parent_name], value=value)
```

Comment:
I am not sure about that, but is `_IMAGES_STACK[ref_parent_name]` really a list?
According to line 121 I would say no, but you must know better ;)

Reply:
No its an unique numpy array, as requested by the first argument of core.substractImages function

Reply:
I was thinking about addition here, now it's corrected I put my comment on the real line